createacls_test.go

```diff
@@ -50,3 +50,37 @@ func TestClientCreateACLs(t *testing.T) {
 		}
 	}
 }
+
+func TestACLPermissionTypeMarshal(t *testing.T) {
+	for i := ACLPermissionTypeUnknown; i <= ACLPermissionTypeAllow; i++ {
```

Comment:
It feels a little weird to hard code the end of the range (`ACLPermissionTypeAllow`) here rather than, say, max(mapping). Is this list not expected to expand?

Reply:
Yeah it's not ideal, but AFAIK there is no clear way to get the minimum and maximum values of an "enum" in Go. This test was taken from Sarama https://github.com/IBM/sarama/blob/82f0e48b0e2b6dfefc3088fe5e45195f6cc461d8/acl_types_test.go#L8